src/pymovements/gaze/experiment.py

```diff
@@ -109,12 +115,15 @@ def __init__(
             origin=origin,
         )
 
-        checks.check_is_not_none(sampling_rate=sampling_rate)
-        assert sampling_rate is not None
+        if sampling_rate is not None:
+            self.sampling_rate = sampling_rate
+        if eyetracker is not None:
+            self.sampling_rate = eyetracker.sampling_rate
```

Comment:
I think we should raise an error that both `eyetracker` and `sampling_rate` is passed.
You can use `checks.check_is_mutual_exclusive()` for that

Reply:
Done